accounts/tests/tests.py

```diff
@@ -0,0 +1,23 @@
+from django.template.loader import get_template
+
+from house.constants import USER_LOGIN_PAGE_ROUTE, USER_SIGNUP_ROUTE
+
+
+class TestLoginPage:
+    def test_user_login_template(self):
+        get_template(USER_LOGIN_PAGE_ROUTE)
```

Comment:
Doesn't this function miss assert?

Reply:
Nope, If the get template will fail the test will fail with a message of the Exception

Reply:
Will it show up on the test description when you run pytest?